### PR TITLE
Kubernetes dashboard tab renaming

### PR DESCRIPTION
**Pull request by jsladerman**
Renamed and refactored accordingly:
Discovery -> Network
Access -> RBAC

### CODE REVIEW

File: assets/src/components/kubernetes/rbac/RBAC.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const directory = [
   { path: SERVICE_ACCOUNTS_REL_PATH, label: 'Service Accounts' },
 ] as const
 
-export default function Access() {
+export default function RBAC() {
```
Comment (michaeljguarino) [inline assets/src/components/kubernetes/rbac/RBAC.tsx:41]: I'm not sure the convension around casing these acronyms, but i feel like `Rbac` and `Rbac.tsx` seems more correct perhaps?

Comment (jsladerman) [reply assets/src/components/kubernetes/rbac/RBAC.tsx]: fixed

Comment (michaeljguarino) [reply assets/src/components/kubernetes/rbac/RBAC.tsx]: I think the filename is still `RBAC.tsx` though

Comment (jsladerman) [reply assets/src/components/kubernetes/rbac/RBAC.tsx]: Yeah just realized git ignored the casing change. fixed

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**